app/portainer/components/datatables/components/TableHeaderRow.tsx

```diff
@@ -26,13 +26,21 @@ export function TableHeaderRow<
           headerProps={{
             ...column.getHeaderProps({
               className: column.className,
+              style: {
+                width:
+                  column.id === 'selection' || column.id === 'actions'
```

Comment:
then we can just apply `column.width` if it's there, no? without checking for specific id. I see a future where this list of ids is very long

Reply:
The problem with always using `column.width` is that the `column` object always comes with a value for it, wdyt about changing the specific id lookup for `column.disableResizing ? column.width : '',`, that way we can manipulate this from any particular column definition, and also the `useRowSelectColumn` library sets that to `true` internally

Reply:
looks good